docs/en/edge/guide/platforms/android/index.md

```diff
@@ -29,11 +29,12 @@ Overview.  For details on the CLI, see The Command-Line Interface.
 
 ## Requirements and Support
 
-Cordova for Android requires the Android SDK. See the Android SDK's
-[System Requirements](http://developer.android.com/sdk/index.html).
+Cordova for Android requires the Android SDK which could be installed
+on OS X, Linux or Windows operation system. See the Android SDK's
+[System Requirements](http://developer.android.com/sdk/index.html#Requirements).
 
 Cordova supports Android 2.3.x (Gingerbread, starting with Android API level 10)
```

Comment:
Thx Steve, will double check, but I still see minSdkVersion is 10 in platform template
 < uses-sdk android:minSdkVersion="10" />